bin/templates/scripts/cordova/lib/prepare.js

```diff
@@ -275,12 +275,54 @@ function handleOrientationSettings (platformConfig, infoPlist) {
     }
 }
 
+// Make sure only update properties from our target project
+function updateBuildPropertyLocal (proj, targetName, prop, value, build) {
+    try {
+        // Check if we have a valid target - during prepare we do not have it
+        var target = proj.pbxTargetByName(targetName);
+        if (target == null || target.buildConfigurationList == null) {
+            proj.updateBuildProperty(prop, value, build);
+        } else {
+            var targetProjectBuildReference = target.buildConfigurationList;
+            // Collect the uuid's from the configuration of our target
+            var COMMENT_KEY = /_comment$/;
+            var validConfigs = [];
+            var configList = proj.pbxXCConfigurationList();
```

Comment:
Try to use `const` and `let` here as well.. any anywhere else..

Reply:
Just a nitpick but `targetProjectBuildReference` is used only once and also it is 2 characters less than `target.buildConfigurationList`, you might not actually need to create an extra variable.

Reply:
Done